visualization/app/codeCharta/ui/ribbonBar/areaSettingsPanel/invertAreaOption/invertAreaOption.component.ts

```diff
@@ -1,15 +1,15 @@
 import "./invertAreaOption.component.scss"
-import { Component, Inject, OnInit } from "@angular/core"
-import { Store } from "../../../state/angular-redux/store"
-import { invertAreaSelector } from "../../../state/store/appSettings/invertArea/invertArea.selector"
+import { Component, Inject } from "@angular/core"
+import { Store } from "../../../../state/angular-redux/store"
+import { invertAreaSelector } from "../../../../state/store/appSettings/invertArea/invertArea.selector"
+import { toggleInvertingArea } from "../../../../state/store/appSettings/invertArea/invertArea.actions"
 import { Observable } from "rxjs"
-import { toggleInvertingArea } from "../../../state/store/appSettings/invertArea/invertArea.actions"
 
 @Component({
 	selector: "cc-invert-area-option",
 	template: require("./invertAreaOption.component.html")
 })
-export class InvertAreaOptionComponent implements OnInit {
+export class InvertAreaOptionComponent {
```

Comment:
this still does implement `OnInit` - depending on compile process this actually may lead to ngOnInit not being executed.

Reply:
Resolved in the next commits